### ISSUE TITLE
Stryker and Mocha Integration

### ISSUE THREAD

**Issue by gohackfelipe** [stryker-mutator/stryker#1624]
### Summary

Hi all,

I'm new at stryker and I've tried to build a POC so that i learn about mutation tests.

My project have the follow structure:

![image](https://user-images.githubusercontent.com/15708865/60773214-b6b1da80-a0d8-11e9-948d-c9141c1826a0.png)

However when I run the stryker it says 'Mocha test run completed: 0/0 passed'. I'd like to understand the reason about it.

### Source Code

src/*/app.js

```js
class Calculator {
  constructor(x, y) {
    this.total = 0;
    this.x = x;
    this.y = y;
  }

  sum() {
    return this.x + this.y;
  }
}

module.exports = Calculator;
```
test/*/app.js

```js
var assert = require('assert');
var Calc = require('../../../src/calculator/app');


describe('Calculator', function() {
  describe('sum', function() {
    it('should return 2', function() {
      assert.equal(2, new Calc(1,1).sum());
    });
  });
});
```

### Stryker config

```js
module.exports = function(config) {
  config.set({
    mutator: "javascript",
    packageManager: "npm",
    logLevel : 'debug',
    files: ['src/**/*.js', '!src/**/index.js', 'test/**/*.js'],
    mutate:[
      "src/calculator/app.js"
    ],
    reporters: ["html", "clear-text", "progress"],
    testRunner: "mocha",
    transpilers: [],
    testFramework: "mocha",
    coverageAnalysis: "perTest",
    mochaOptions: {
      files: [ 'test/unit/calculator/app.js' , "src/calculator/app.js"],
      asyncOnly: false,
    }
  });
};
}
```

### Stryker environment

```
+-- "@stryker-mutator/core": "^2.0.1",
+-- "@stryker-mutator/html-reporter": "^2.0.1",
+-- "@stryker-mutator/javascript-mutator": "^2.0.1",
+-- "@stryker-mutator/mocha-framework": "^2.0.1",
+-- "@stryker-mutator/mocha-runner": "^2.0.1",
+-- "stryker-mocha-framework": "^0.15.1"
```

### Test runner environment

```
styker run

```

### Your Environment

| software         | version(s)
| ---------------- | -------  
| node             |  v10.11.0
| npm              |  6.10.0
| Operating System | Mac OS X 10.14.4

### Add stryker.log

[stryker.log](https://github.com/stryker-mutator/stryker/files/3366067/stryker.log)

**Comment by nicojs** [comment]
Could you try this?

```ts
module.exports = function(config) {
  config.set({
    mutator: "javascript",
    packageManager: "npm",
    logLevel : 'debug',
    files: ['src/**/*.js', '!src/**/index.js', 'test/**/*.js'],
    mutate:[
      "src/calculator/app.js"
    ],
    reporters: ["html", "clear-text", "progress"],
    testRunner: "mocha",
    transpilers: [],
    testFramework: "mocha",
    coverageAnalysis: "perTest",
    mochaOptions: {
      spec: [ 'test/unit/calculator/app.js' ],
      asyncOnly: false,
    }
  });
};
}
```

`files` was renamed to `spec` to align with mocha's options in the 2.0 release of Stryker. Where did you find the reference to `files`? We might need to update more documentation.

**Comment by gohackfelipe** [comment]
Hi @nicojs , thanks for you answer !
After change to 'spec' it works. However I received another error.

It looks like a tentative to delete the temp folder. But stryker cannot .

[stryker.log](https://github.com/stryker-mutator/stryker/files/3368245/stryker.log)

**Comment by nicojs** [comment]
Yeah, we have some trouble with the cleanup of our child processes. That should be fixed with #1534 . I'll do a release later today so you can test it.

**Comment by nicojs** [comment]
Released, could you test again with version 2.0.2

**Comment by nicojs** [comment]
Closing this issue for now, as the original problem is fixed. Please open a new issue when the error still happens on 2.0.2
